tests/backup/backup_test.go

```diff
@@ -450,13 +456,13 @@ var _ = Describe("{BackupCrashVolDriver}", func() {
 			}
 		}
 
-		Step("teardown all applications on source cluster before switching context to destination cluster", func() {
+		/*Step("teardown all applications on source cluster before switching context to destination cluster", func() {
```

Comment:
Why don't we tear down apps at this point?

Reply:
Temporarily I have disabled deletion of apps on source cluster. Will re-enable them and remove also some of the debug logs before merging.

Reply:
Uncommented with latest commit